consensus/common_test.go

```diff
@@ -72,16 +74,20 @@ func NewValidatorStub(privValidator types.PrivValidator, valIndex int) *validato
 }
 
 func (vs *validatorStub) signVote(voteType types.SignedMsgType, hash []byte, header types.PartSetHeader) (*types.Vote, error) {
+	addr, err := vs.PrivValidator.GetAddress()
+	if err != nil {
+		return nil, errors.Wrap(err, "Error while retrieving private validator's address")
```

Comment:
better to use single style everywhere

```suggestion
		return nil, errors.Wrap(err, "failed to get address from private validator")
```